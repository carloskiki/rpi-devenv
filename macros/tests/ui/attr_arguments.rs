#[macros::main(name = "kernel_main")]
pub fn kernel_main() -> ! {}

fn main() {}
