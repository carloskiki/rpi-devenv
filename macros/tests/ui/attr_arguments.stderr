error: this macro takes no arguments
 --> tests/ui/attr_arguments.rs:1:16
  |
1 | #[macros::main(name = "kernel_main")]
  |                ^^^^^^^^^^^^^^^^^^^^
